corpus/arguments.txt

```diff
@@ -1,31 +1,52 @@
 ================================================================================
-Simple argument
+simple argument
 ================================================================================
 This in an argument: {arg}
 --------------------------------------------------------------------------------
 
 (help_file
-  (line
-    (word)
-    (word)
-    (word)
-    (word)
-    (argument
-      (word))))
+  (block
+    (line
+      (word)
+      (word)
+      (word)
+      (word)
+      (argument
+        (word)))))
 
 ================================================================================
-Multiple arguments on the same line
+multiple arguments on the same line
 ================================================================================
-
 {foo} {bar} {baz}
 
 --------------------------------------------------------------------------------
 
 (help_file
-  (line
-    (argument
-      (word))
-    (argument
-      (word))
-    (argument
+  (block
+    (line
+      (argument
+        (word))
+      (argument
+        (word))
+      (argument
+        (word)))))
+
+================================================================================
+NOT an argument
+================================================================================
+{foo "{bar}" `{baz}` |{baz| }
+
+--------------------------------------------------------------------------------
+
+(help_file
+  (block
+    (line
+      (argument
+        (word)
+        (ERROR))
```

Comment:
I gave up on this for now, ideally it should be:

      (argument
        (word)
        (MISSING "}"))